### PR TITLE
log: support change the level of etcd for debugging

### LINKED ISSUE #974: support change the level of etcd for debugging
## Feature Request

**Is your feature request related to a problem? Please describe:**

    If etcd started failed, we may not know the reason because we only logging error level log.

**Describe the feature you'd like:**

    support to change the level of etcd log with/without adding any new config items (or output the log to other files).

    NOTE: always using `info` level often has too many logs.

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

### PR DESCRIPTION
**Pull request by GMHDBJD**
### What problem does this PR solve? 
close #974 

### What is changed and how it works?
- set etcd-log-level to `warn` if log-level is `info`, otherwise set it same as log-level

### Check List 

Tests 
 - Manual test (add detailed scripts or steps below)
 Manually check the log file.

### CODE REVIEW

File: dm/master/config.go [hunk 1]
```diff
@@ -439,17 +439,21 @@ func parseURLs(s string) ([]url.URL, error) {
...
 	// NOTE: `genEmbedEtcdConfig` can only be called after logger initialized.
 	// NOTE: if using zap logger for etcd, must build it before any concurrent gRPC calls,
 	// otherwise, DATA RACE occur in NewZapCoreLoggerBuilder and gRPC.
-	// NOTE: we can only increase the log level for the clone logger but not decrease.
-	logger := log.L().WithFields(zap.String("component", "embed etcd")).WithOptions(zap.IncreaseLevel(zap.ErrorLevel))
-	cfg.ZapLoggerBuilder = embed.NewZapCoreLoggerBuilder(logger, logger.Core(), log.Props().Syncer) // use global app props.
+	logger := log.L().WithFields(zap.String("component", "embed etcd"))
+	// if logLevel is info, set etcd log level to WARN to reduce log
+	if strings.ToLower(logLevel) == "info" {
+		logger.Logger = logger.WithOptions(zap.IncreaseLevel(zap.WarnLevel))
```
Comment (lance6716) [inline dm/master/config.go:453]: let's log this change to tell user. and "debug" maybe a level lower than "info"

File: dm/master/config.go [hunk 2]
```diff
@@ -439,17 +439,21 @@ func parseURLs(s string) ([]url.URL, error) {
...
+	}
+
+	cfg.ZapLoggerBuilder = embed.NewZapCoreLoggerBuilder(logger.Logger, logger.Core(), log.Props().Syncer) // use global app props.
```
Comment (lance6716) [inline dm/master/config.go:456]: not sure why change `logger` to `logger.Logger`

**Review by lance6716 — COMMENTED** [review]
rest LGTM

File: dm/master/config.go (hunk 1 above)
Comment (GMHDBJD) [reply dm/master/config.go:454]: add log in 6d12af4
user can use debug level to see more log about etcd.

File: dm/master/config.go (hunk 2 above)
Comment (GMHDBJD) [reply dm/master/config.go:457]: type of logger now is log.Logger after https://github.com/pingcap/dm/blob/7de7bb4fcbe6085cb8f5d7a8fd50589465e7b918/dm/master/config.go#L451 change. The type should be log.Logger.logger

**Comment by lance6716** [comment]
LGTM

**Review by csuzhangxc — APPROVED** [review]
LGTM

**Pull request merged**